gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
-        except AbortionError as exc:
-            if exc.code == StatusCode.ABORTED:
-                raise Conflict(exc.details)
+        except _Rendezvous as exc:
```

Comment:
Don't just make use of the fact that the exception is a `grpc.RpcError` (which you know from your `except` line); also make use of the fact that the exception was raised from calling a `grpc.UnaryUnaryMultiCallable` (and thus [this specification](https://github.com/grpc/grpc/blob/c41f81aeeb054ec436934c7cc1afeb1dee0a8581/src/python/grpcio/grpc/__init__.py#L473) also applies and is available for your use).